testing/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "vote.gov-cypress-testing-suite",
+  "version": "1.0.0",
+  "description": "## Requirements",
+  "main": "index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "axe-core": "^4.6.3",
+    "cypress": "^12.3.0",
+    "cypress-axe": "^1.3.0"
+  },
+  "scripts": {
+    "cy:open": "cypress open",
+    "cy:test": "cypress run --spec 'cypress/e2e/tests/**/*'",
```

Comment:
so i had a `cy.test` already but i think that one just came with the file when I ran the `npm init` command.  i took out the other `test` in the file but here is the `cy:test` that will run the cypress test headless.